client/scss/components/_footer.scss

```diff
@@ -166,3 +166,58 @@
     }
   }
 }
+
+// Footer control bar for performing actions on the page
+footer .actions,
+footer .preview {
```

Comment:
Any reason we aren't using classes here? Maybe add a new `w-footer` one?

Reply:
This is just moving existing code around, I haven’t changed this component at all.